tests/unit/test_config.py

```diff
@@ -172,14 +171,6 @@ def test_azure_config(workspace: Workspace) -> None:
         "embedding_model_deployment_id": "embedding-deployment-id-for-azure",
     }
 
-    assert openai.api_type == config.openai_api_type
-    assert openai.api_base == config.openai_api_base
-    assert openai.api_version == config.openai_api_version
-
-    # Reset config
-    os.environ["USE_AZURE"] = "False"
```

Comment:
This doesn't do anything.  Updating the config does not update the environment variables, so we don't need to reset it.